client/pluginmanager/csimanager/interface.go

```diff
@@ -17,9 +18,31 @@ type MountInfo struct {
 	IsDevice bool
 }
 
+type UsageOptions struct {
+	ReadOnly       bool
+	AttachmentMode string
+	AccessMode     string
+}
+
+func (u *UsageOptions) ToFS() string {
```

Comment:
It looks like this only ever gets used as a file path segment. Can we add a docstring here about that intended use so that future developers never change this to something annoying to handle? (ex. prefixed by `-`)